package.json

```diff
@@ -3,6 +3,9 @@
   "version": "21.0.1",
   "description": "An ESLint Shareable Config for JavaScript Standard Style with TypeScript support",
   "main": "lib/index.js",
+  "engines": {
+    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
+  },
```

Comment:
It is what ESLint >= 8.0.0 support:
Ref: https://github.com/eslint/eslint/blob/6f940c3ce715327f282c197d0f71b91848e5d83d/package.json#L146-L148

Reply:
Still seems like an unrelated change, doesn't it? Wouldn't this PR be complete without it?

Reply:
Yes, we can merge the PR without this, but at least it would be clearer for users what Node.js version we support.
I can revert this change if you really want to.

Reply:
Yes, please.

Reply:
Updated. :+1: